backend/main.py

```
@@ -1608,5 +1683,176 @@ def send_status_notification(issue_id: int, old_status: str, new_status: str, no
+            escalation_rate=escalation_rate
+        )
+
+    except Exception as e:
+        logger.error(f"Error getting escalation stats: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail="Failed to retrieve escalation statistics")
+
+@app.post("/api/grievances/{grievance_id}/escalate")
+def manual_escalate_grievance(
+    grievance_id: int,
+    reason: str = Query(..., description="Reason for manual escalation"),
+    db: Session = Depends(get_db)
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)